lib/index.js

```diff
@@ -104,7 +104,9 @@ const setup = ({customTextFormattings, parseFigureProps, customCaption} = {}) =>
       assert(oldArticleElm, 'oldArticleElm must exists');
       assert(newArticleElm, 'newArticleElm must exists');
 
-      patchDom({oldArticleElm, newArticleElm});
+      if (oldArticleElm.innerHTML !== newArticleElm.innerHTML) {
```

Comment:
nitpick, innerHTML can potentially be slow for large documents, not sure if it affects anything though